apps/cowswap-frontend/src/common/hooks/useMenuItems.ts

```diff
@@ -1,14 +1,25 @@
 import { useMemo } from 'react'
 
 import { useFeatureFlags } from '@cowprotocol/common-hooks'
-import { isLocal } from '@cowprotocol/common-utils'
+import { isLocal, isPr } from '@cowprotocol/common-utils'
 
-import { HOOKS_STORE_MENU_ITEM, MENU_ITEMS } from '../constants/routes'
+import { HOOKS_STORE_MENU_ITEM, MENU_ITEMS, YIELD_MENU_ITEM } from '../constants/routes'
 
 export function useMenuItems() {
   const { isHooksStoreEnabled } = useFeatureFlags()
+  const { isYieldEnabled } = useFeatureFlags()
```

Comment:
Should this also be used for the form itself?

Reply:
I'm not sure, we also don't hide Hooks widget iteself, only the link to it.
So, I don't have a strong opinion on that.
Please, ping if you think we should hide the form as well.